app/validation/error_messages.py

```diff
@@ -9,5 +9,5 @@
     'INVALID_DATE_RANGE_TO_BEFORE_FROM': "The 'period to' date cannot be before the 'period from' date.",
     'INVALID_DATE_RANGE_TO_FROM_SAME': "The 'period to' date must be different to the 'period from' date.",
     'MAX_LENGTH_EXCEEDED': "Your answer is too long, it has to be less than %(max)d characters",
-    'INVALID_DECIMAL': "Please only enter numbers to %(max)d decimal places into the field.",
+    'INVALID_DECIMAL': "Please only enter whole numbers into the field.",
```

Comment:
This copy change doesn't make sense for situations where there are decimal places required. The only reason it passes the functional tests is due to the error message being overridden for the test field, which I doubt we want to force authors to do for every decimal field that appears in a survey.

I suggest that we add an `INVALID_INTEGER` error message and adjust the code accordingly. We should also remove the overridden `INVALID_DECIMAL` error message from `test_numbers.json`